_blog-src/_posts/2016-01-26-export-import.markdown

```diff
@@ -0,0 +1,155 @@
+
+---
+layout: post
+title:  "Apiman 1.2.1 Export and Import"
+date:   2016-01-26 20:30:00
+author: len_dimaggio
+categories: apiman
+---
+
+### The Question you Dread
+
+If you use a computer at home or at work, you'll eventually find yourself in a situation where you lose some important data and, while you are trying to recover it, someone asks you a question that is simultaneously annoying and terrifying:
+
+*"Did you make a backup?"*
+
+Happily, the 1.2 release of apiman includes a new feature that enables you to export and import your apiman data and provides you with an easy means to create apiman data backups.  In this post,  we'll take a look at the new export/import feature, and how you can use it for a variety of tasks to protect your data, make your life easier, and enable you to avoid annoying and terrifying questions.
+
+For test data, we'll use the same types of data (organizations, users, plans, policies, APIs and client apps), that we created in the initial post in this series. (  [The Impatient New User Guide to apiman](http://www.apiman.io/blog/introduction/overview/2015/01/09/impatient-new-user-redux.html) )
+
+Note that since that post was written in January 2015, some of the names of the data elements have changed. You can either create the test data referred to in this post yourself, or you can import the data file attached to the post.
+
```

Comment:
You need to add a section as follows:

```
////
Use this pattern to avoid asciidoctor rendering the HTML comment.
<!--more-->
////
```

To generate a preview snippet - otherwise the entire article is rendered on the blog index pages